app/src/androidTest/java/com/github/se/project/ui/lesson/TutorLessonResponseTest.kt

```diff
@@ -129,7 +129,7 @@ class TutorLessonResponseTest {
       TutorLessonResponseScreen(
           mockListProfilesViewModel, mockLessonViewModel, mockNavigationActions)
     }
-    Thread.sleep(5000)
+    composeTestRule.waitForIdle()
```

Comment:
Use waitForIdle instead of sleep is much more revelant to avoid wasting time and to be sure the necessary time is given! Good job!